spec/requests/admin/lead_providers/lead_provider_users_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Admin::LeadProviders::LeadProviderUsers", type: :request do
+  let(:lead_provider_profile) { create(:lead_provider_profile) }
+  let(:lead_provider_user) { lead_provider_profile.user }
+
+  before do
+    admin_user = create(:user, :admin)
```

Comment:
You could put it to let and just leave admin user

Reply:
Indeed. I left it as it's only used in this one place so I didn't see any use in having it as a variable

Reply:
:) But it did end up as variable - plus there is a level of elegance of having variables in lets and grouping them together - not hard rule - I'll leave the decision to you. Just it looks nicer when you have all your declarations in lets

Reply:
Personally I find it more elegant to have variables that are as local as possible. I would expect things in `let` to be used in more than one place, precisely because that's the power of `let`s. Kind of how I would not make a variable an `@`-variable, unless I have to.